### ISSUE TITLE
Shared Tslint config

### ISSUE THREAD

**Issue by jhuntoo** [CitadelJS/core#12]
Keep it in @citadel/tools

It gets merged into repos via git with a tagged version perhaps.

**Comment by lholznagel** [comment]
Some more Information:
so with some researching and asking @otbe and I found out that this is currently the best way to keep our configs up to date in all repos
The commands are like follows:
```
git remote add config git@github.com/CitadelJS/tools
git fetch config
git merge --squash config/master
git commit -m "updated configs to vX.Y.Z" // maybe update on every commit
git push
```
This script should be triggered on every commit that goes to the repo `configs`

General question:
how about renaming tools to tooling?

**Comment by otbe** [comment]
@lholznagel 
```@citadel/config``` repo is already there 😁

**Comment by lholznagel** [comment]
yeah but I thought only configs like tsconfig, tslint come in there and scripts like the updater go into a separate repo, because the script does not have to be in every repo

**Comment by otbe** [comment]
For sure! Misread your comment somehow. Sorry!

**Comment by lholznagel** [comment]
Okay, everything good :)

**Comment by lholznagel** [comment]
Think that this can be closed. This is fixed with citadeljs/tooling#1 and citadeljs/tooling#2
